rpcclient/wsclient/wsclient.go

```diff
@@ -20,9 +20,8 @@ type WsClient interface {
 	Close()
 	Discover(provides types.ProvidesCoin, searchTime uint64) ([][]string, error)
 	Query(maddr string) (*rpctypes.QueryPeerResponse, error)
-	SubscribeSwapStatus(id uint64) (<-chan types.Status, error)
-	TakeOfferAndSubscribe(multiaddr, offerID string,
-		providesAmount float64) (id uint64, ch <-chan types.Status, err error)
+	SubscribeSwapStatus(id types.Hash) (<-chan types.Status, error)
+	TakeOfferAndSubscribe(multiaddr, offerID string, providesAmount float64) (ch <-chan types.Status, err error)
```

Comment:
I didn't change these functions.  The interface signatures were just out of date with the existing implementation.